Mathlib/Data/Finsupp/Lex.lean

```diff
@@ -104,6 +104,25 @@ theorem lt_of_forall_lt_of_lt (a b : Lex (α →₀ N)) (i : α) :
     (∀ j < i, ofLex a j = ofLex b j) → ofLex a i < ofLex b i → a < b :=
   fun h1 h2 ↦ ⟨i, h1, h2⟩
 
+theorem lex_lt_iff {a b : Lex (α →₀ N)} :
+    a < b ↔ ∃ i, (∀ j, j< i → ofLex a j = ofLex b j) ∧ ofLex a i < ofLex b i :=
+    Finsupp.lex_def
+
+theorem lex_le_iff {a b : Lex (α →₀ N)} :
```

Comment:
I don't think we should have this; it's just the composition of two other basic theorems, and it's not always the most natural characterization (e.g. if `α` and `N` are linear orders, the negation of `lex_lt_iff` would probably be nicer to work with).

Reply:
It was already present for monomial orders (it's where it comes from), but I can delete it.